packages/fxa-auth-server/test/local/customs.js

```diff
@@ -55,7 +54,7 @@ describe('Customs', () => {
   });
 
   it("can create a customs object with url as 'none'", () => {
```

Comment:
The tests were a pain to work with and are using older syntax...battle for another day.